OPHD/UI/FactoryProduction.cpp

```diff
@@ -218,26 +212,24 @@ void FactoryProduction::update()
 
 	Window::update();
 
-	auto& renderer = Utility<Renderer>::get();
-
-	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, const std::string& title, const std::string& text) {
-		renderer.drawText(*FONT_BOLD, title, position, NAS2D::Color::White);
-		position.x() += 120;
-		renderer.drawText(*FONT, text, position, NAS2D::Color::White);
-	};
-
+	const int labelWidth = 120;
 	auto position = mRect.startPoint() + NAS2D::Vector{constants::MARGIN * 2 + static_cast<int>(mProductGrid.width()), 25};
-	drawTitleText(position, "Turns Completed:", std::to_string(mFactory->productionTurnsCompleted()) + " of " + std::to_string(mProductCost.turnsToBuild()));
+	auto value = std::to_string(mFactory->productionTurnsCompleted()) + " of " + std::to_string(mProductCost.turnsToBuild());
+	drawLabelAndValueLeftJustify(position, "Turns Completed:", value, 120);
```

Comment:
Passing an explicit `120` rather than `labelWidth`.

Reply:
Thanks for catching my sloppiness here